### PR TITLE
Fix lineendings

### PR DESCRIPTION
**Pull request by incaseoftrouble**
I added a `.gitattributes` file as suggested by @blacelle in https://github.com/RoaringBitmap/RoaringBitmap/commit/7c1e4a398d9e107bd6d5e45eebbf813cceef0bce and ran `git add --renormalize` as suggested by the [git docs](https://git-scm.com/docs/gitattributes)

### CODE REVIEW

**Comment by lemire** [comment]
@incaseoftrouble Can you sync with master now that the PR on ArrayContainer PR was pulled in relation with https://github.com/RoaringBitmap/RoaringBitmap/issues/313

I could do it but I think that we are less likely to encounter a problem if the same person fixes all the line endings (presumably using the same approach throughout).

**Comment by incaseoftrouble** [comment]
Done. As indicated by the diff ( https://github.com/RoaringBitmap/RoaringBitmap/pull/315/files?w=1 ) there are only whitespace changes aside from the addition of `.gitattributes`

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Set the default behavior, in case people don't have core.autocrlf set.
+* text=auto eol=lf
```
Comment (blacelle) [inline .gitattributes]: I suggest leaving text=auto and not eol=lf

People working under Windows might get into trouble with this.

Comment (incaseoftrouble) [reply .gitattributes]: Done. Wasn't sure which is better.

**Comment by lemire** [comment]
I have verified that the changes are white-space only. Merging.

**Pull request merged**